utils/data_processing/csv2json.py

```diff
@@ -108,13 +96,17 @@ def main(infile, outfile, header, units, coded, response, title, doi, source,
     # Loading file to a dataframe to keep column name
     df = pd.read_csv(
         # FIXME problem with the encoding for variables containing greek letters
-        infile, header=header_value, index_col=None, encoding="utf-8", sep=";", decimal=decimal
+        infile,
+        header=header_value,
+        index_col=None,
+        encoding="windows-1252",
+        sep=";",
```

Comment:
Is this adding the changes @saif-analyze needed to convert his experiments?

Reply:
Not yet, I need to add another option to do this